infinispan-remote/src/test/java/org/hibernate/ogm/datastore/infinispanremote/utils/InfinispanRemoteTestHelper.java

```diff
@@ -65,7 +65,7 @@ public long getNumberOfAssociations(SessionFactory sessionFactory) {
 	@Override
 	public void dropSchemaAndDatabase(SessionFactory sessionFactory) {
 		final InfinispanRemoteDatastoreProvider datastoreProvider = getProvider( sessionFactory );
-		final Set<String> mappedCacheNames = datastoreProvider.getMappedCacheNames();
+		final Set<String> mappedCacheNames = datastoreProvider.getCacheTemplatesByName().keySet();
```

Comment:
Why don't you keep the other method? It seems easier and clearer.

Reply:
fixed